Tests/WorkspaceTests/WorkspaceTests.swift

```diff
@@ -4905,6 +5172,15 @@ final class WorkspaceTests: XCTestCase {
         )
 
         workspace.checkPackageGraphFailure(roots: ["Root"], forceResolvedVersions: true) { diagnostics in
+            testDiagnostics(diagnostics) { result in
+                result.check(
+                    diagnostic: .prefix(
+                        "a resolved file is required when automatic dependency resolution is disabled and should be placed at \(sandbox.appending(components: "Package.resolved")). Running resolver because the following dependencies were added:"
+                    ),
+                    severity: .error
+                )
+            }
+            /*
```

Comment:
Should this be commented?

Reply:
need to be removed - I improved it ^^. thanks